libraries/rush-lib/src/api/RushConfigurationProject.ts

```diff
@@ -503,3 +508,29 @@ export class RushConfigurationProject {
     return isMain;
   }
 }
+
+export function validateRelativePathField(relativePath: string, name: string): void {
+  if (path.posix.isAbsolute(relativePath) || path.win32.isAbsolute(relativePath)) {
+    throw new Error(`The value "${relativePath}" in the "${name}" field must be a relative path.`);
+  }
+
+  if (relativePath.includes('\\')) {
+    throw new Error(
+      `The value "${relativePath}" in the "${name}" field may not contain backslashes ('\\'), since they are interpreted differently` +
```

Comment:
These errors should probably print which project they're mentioned in, and that it's in rush.json.

Reply:
The value listed can be searched as an exact string and will find the offending value, but I agree with mentioning rush.json.